server/utils/resident/placement.ts

```diff
@@ -0,0 +1,81 @@
+import type { Cas1SpaceBookingShortSummary } from '@approved-premises/api'
+import { SummaryListWithCard } from '@approved-premises/ui'
+import { summaryListItem, summaryListItemNoBlankRows } from '../formUtils'
+import paths from '../../paths/manage'
+import { card, PlacementSubTab } from './index'
+
+export const placementSideNavigation = (subTab: PlacementSubTab, crn: string, placementId: string) => {
+  return [
+    {
+      text: 'Placement details',
+      href: paths.resident.tabPlacement.placementDetails({ crn, placementId }),
+      active: subTab === 'placement-details',
+    },
+    {
+      text: 'Application',
+      href: paths.resident.tabPlacement.application({ crn, placementId }),
+      active: subTab === 'application',
+    },
+    {
+      text: 'Previous AP stays',
+      href: paths.resident.tabPlacement.previousApStays({ crn, placementId }),
+      active: subTab === 'previous-ap-stays',
+    },
+  ]
+}
+
+// TODO: Some of these empty sections [] will be filled in the next relevant PRs/tickets
```

Comment:
Probably not a good plan to add extra empty sections then I'd suggest. It makes it likely your PR will conflict more with Rob's for instance as he's working on these empty tabs. Probably best to remove them.